source/unit.js

```diff
@@ -581,9 +586,106 @@ export class Unit {
     // divided by any effects of prefixes applied to the "to" unit, which
     // is assumed to be some form of a gram unit
     return massAmt / massUnit.magnitude_ ;
-  }
+  } // end convertMolToMass
+
+  /**
+   * Converts equivalents to mass.
+   * 
+   * @param {number} equivalents - The amount in equivalents to be converted.
+   * @param {object} targetUnit - The target/to unit for which the converted number is wanted.
+   * @param {number} molecularWeight - The molecular weight of the substance for which the conversion is being made.
+   * @param {number} charge - The charge of the substance for which the conversion is being made.
```

Comment:
I think you mean "absolute value of the charge", here and in other comments below.